src/Liuggio/StatsdClient/Service/StatsdService.php

```diff
@@ -82,7 +82,7 @@ public function getSamplingRate()
     public function timing($key, $time)
     {
         $this->appendToBuffer(
-            $this->factory->timing($key, $time)
+            $this->factory->timing($this->getKeyMetric($key), $time)
```

Comment:
Please move prefixing/suffixing logic within the StatsdDataFactory::produceStatsdData function. It is a bit sorry to delegate this logic here, because it won't benefit other classes easily.